config/co2eq_parameters_lifecycle.json

```diff
@@ -2937,6 +2937,12 @@
           "value": 17
         }
       },
+      "FO-NI": {
```

Comment:
You can also put 0 for direct, otherwise it will fallback to the default direct unknown.

Reply:
Will do!